core/src/test/resources/org/apache/calcite/test/RelOptRulesTest.xml

```diff
@@ -1990,7 +2018,8 @@ LogicalProject(EXPR$0=[+($1, $5)], EXPR$1=[SUM($6) OVER (PARTITION BY $4 RANGE B
     </TestCase>
     <TestCase name="testPushProjectPastSetOp">
         <Resource name="sql">
-            <![CDATA[select sal from (select * from emp e1 union all select * from emp e2)]]>
+            <![CDATA[select sal from
+(select * from emp e1 union all select * from emp e2)]]>
         </Resource>
```

Comment:
These changes seem not necessary, can you revert it back ?

Reply:
I have updated this file as it was recommended in this comment: https://issues.apache.org/jira/browse/CALCITE-3393?focusedCommentId=16958082&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-16958082. But if you insist, I can leave only changes required for my tests.

Reply:
Thanks, let's keep it as it is now.